content/developer/tutorials/getting_started/07_basicviews.rst

```diff
@@ -126,7 +126,8 @@ be convenient to use the ``--dev xml`` parameter when launching the server:
 
 .. code-block:: console
 
-    $ ./odoo-bin --addons-path=addons,../enterprise/,../technical-training-sandbox/ -d rd-demo -u estate --dev xml
+    $ pip install inotify
```

Comment:
Not sure if it's the right place (`--dev xml` only) and manner to introduce `inotify`

Reply:
Alright, just thought of it because one of the newbies didn't install `inotify` and didn't noticed his views were not correctly updated, he continued the tutorial then I had to fix a lot of errors he made and didn't noticed because it was not properly reloading.

Reply:
`inotify` is only required for reloading the server when a python file is updated. `--dev xml` works just fine without it, so I wouldn't mention that package here.